common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1471,65 +1500,32 @@ def clean_id(self, padding_char='='):
         """
         return course_metadata_utils.clean_course_key(self.location.course_key, padding_char)
 
-    @property
-    def teams_conf(self):
-        """
-        Returns a TeamsConfig object wrapping the dict `teams_configuration`.
-
-        TODO: In MST-18, `teams_configuration` will be a custom field that
-        parses its input into a TeamsConfig object.
-        All references to this property will become references to
-        `.teams_configuration`.
-        """
-        # If the `teams_configuration` dict hasn't changed, return a cached
-        # `TeamsConfig` to avoid re-parsing things.
-        # If it has changed, recompute the `TeamsConfig`.
-        try:
-            cached_teams_conf, cached_teams_conf_source = (
-                self._teams_conf, self._teams_conf_source
-            )
-        except AttributeError:
-            pass
-        else:
-            if cached_teams_conf_source == self.teams_configuration:
-                return cached_teams_conf
-        self._teams_conf_source = self.teams_configuration
-        self._teams_conf = TeamsConfig(self._teams_conf_source)
-        return self._teams_conf
-
     @property
     def teams_enabled(self):
```

Comment:
If this is also a passthrough, why not also remove this property just like `teams_topics` and `teams_max_size`?

Reply:
Could definitely remove it. My thought was that it keeps that uses it more brief, and keeps this diff smaller :)